templates/cpp-template-default/proj.android-studio/gradle.properties

```diff
@@ -16,10 +16,18 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-PROP_COMPILE_SDK_VERSION=14
-PROP_MIN_SDK_VERSION=10
-PROP_TARGET_SDK_VERSION=22
-PROP_APP_PLATFORM=10
+
+# Android SDK version that will be used as the compile and target version of this project
+PROP_SDK_VERSION=27
+
+# Android SDK version that will be used as the earliest version of android this application can
+# run on. Targeting Android 16 (Jellybean) this that support 99% of current android devices
+PROP_MIN_SDK_VERSION=16
```

Comment:
Should the min sdk version be `14` for more reasonable?

Reply:
If could be. Android SDK says it's obsolete so I thought hitting 99% of current android devices would suffice

https://developer.android.com/about/dashboards/index.html

Reply:
Yep, but probably Google will not collect the all information of android devices since there are lots of Phone remove Google Service, especially in China.
So if it could be lower version, let's keep the lower version.

Reply:
Ok! will lower to `14`

Reply:
Thanks. You could refer to the page you refered: https://medium.com/google-developers/picking-your-compilesdkversion-minsdkversion-targetsdkversion-a098a0341ebd

> Of course, if a new API is key to your entire app, then that makes the minSdkVersion discussion quite a bit easier. Just remember that even 0.7% of 1.4 billion devices is a lot of devices.

Reply:
Very true! Updating now